### PR TITLE
[DDP-8388] add mbc & brain new medications

### PR DESCRIPTION
**Pull request by LKapanad**
[DDP-8388] [DDP-8386] add medications for mbc, brain & angio
drugs list is fetched from /suggestions/drugs endpoint which is common for these studies.

[DDP-8388]: https://broadinstitute.atlassian.net/browse/DDP-8388?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DDP-8386]: https://broadinstitute.atlassian.net/browse/DDP-8386?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by zbokucha — DISMISSED** [review]

File: pepper-apis/dsm-core/src/main/resources/liquibase/051-updateDB.xml [hunk 1]
```diff
@@ -0,0 +1,586 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/resources/liquibase/051-updateDB.xml]: Can we please change the file name from `051-updateDB.xml` to `DDP-8388_drug-list-update.xml`  and also move it into the CMI folder under liquibase

File: pepper-apis/dsm-core/src/main/resources/master-changelog.xml [hunk 2]
```diff
@@ -52,6 +52,7 @@
     <include file="liquibase/048-updateDB.xml" relativeToChangelogFile="true"/>
     <include file="liquibase/049-updateDB.xml" relativeToChangelogFile="true"/>
     <include file="liquibase/050-updateDB.xml" relativeToChangelogFile="true"/>
+    <include file="liquibase/051-updateDB.xml" relativeToChangelogFile="true"/>
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/resources/master-changelog.xml]: Needs to get changed to the new file name

Comment (SimoneMaiwald) [reply pepper-apis/dsm-core/src/main/resources/master-changelog.xml]: and added to the end of the list please

File: pepper-apis/dsm-core/src/main/resources/liquibase/CMI/DDP-8388_drug-list-update.xml [hunk 3]
```diff
@@ -0,0 +1,586 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="DDP-8388_druglist" author="LKapanad">
+        <insert tableName="drug_list">
+            <column name="generic_name" type="varchar(200)" value="AMIVANTAMAB"/>
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/resources/liquibase/CMI/DDP-8388_drug-list-update.xml]: as far as I know we don't need to add the `type` to the add column statements. would make the whole thing better readable

**Review by SimoneMaiwald — APPROVED** [review]

**Pull request merged**